tests/hpc/barrier_init.pm

```diff
@@ -22,11 +22,12 @@ sub run {
     my $nodes = get_required_var("CLUSTER_NODES");
 
     # Initialize barriers
-    if (check_var("HPC", "slurm")) {
-        barrier_create("SLURM_MASTER_SERVICE_ENABLED", $nodes);
-        barrier_create("SLURM_SLAVE_SERVICE_ENABLED",  $nodes);
-        barrier_create("SLURM_SETUP_DONE",             $nodes);
+    if (check_var('HPC', 'slurm')) {
```

Comment:
It would be nice if you would use the same quotation style in all checks

Reply:
done